src/core/agent/services/signifyNotificationService.ts

```diff
@@ -202,6 +202,10 @@ class SignifyNotificationService extends AgentService {
           .submitAdmit(ourIdentifier.signifyName, admit, sigs, aend, [
             exchange.exn.i,
           ]);
+        await Agent.agent.ipexCommunications.createLinkedIpexMessageRecord(
```

Comment:
This specifically means they presenting the credential again, usually to update the revocation status.

Reply:
If we receive IPEX grant but `existingCredential` is false - this is an issuance, we need to store it at that point. Not when we call `admitAcdc` in `connectionService`.

If we receive an `IpexApply`, we store the record immediately.